src/main/resources/META-INF/reaktivity/core.idl

```diff
@@ -73,6 +97,8 @@ scope core
 
         struct Begin extends core::stream::Frame [0x00000001]
         {
+            int64 authMask;
+            int64 authExpires;
```

Comment:
I think we should remove this and have the security nukleus deal with expiration